NASA-AMMOS/aerie-ui
# Units UI

resolves #822 
depends on https://github.com/NASA-AMMOS/aerie/pull/1055

Figma: https://www.figma.com/file/sqQoaecl7WgCFFyIC4KnK2/Aerie-Units-and-Legends?node-id=22%3A9465&mode=dev

To test:
1. Create a plan using the updated banananation-develop.jar file
2. Add `BakeBananaBread` and `BiteBanana` activities
3. Verify that selecting these added activities, the arguments in the Selected Activity panel displays the units
4. Run the simulation
5. Select the `BiteBanana` span
6. Verify that the parameters have units
